DependencyInjection/Configuration.php

```diff
@@ -581,10 +581,10 @@ private function createCachePluginNode()
 
                             return $v;
                         })
-                        ->end()
+                    ->end()
                     ->validate()
-                    ->ifNotInArray([null, true, false])
-                        ->thenInvalid('Value for "respect_cache_headers" must be null or boolean')
+                        ->ifNotInArray([null, true, false])
+                            ->thenInvalid('Value for "respect_cache_headers" must be null or boolean')
```

Comment:
`ifNotInArray` does not create a new node in the tree (otherwise it would need an end node too). It configures the validation node. So this one should be unindented (it was right before your last update)